components/mediators/cache/org.wso2.carbon.mediator.cache/src/main/java/org/wso2/carbon/mediator/cache/CacheMediatorFactory.java

```diff
@@ -344,6 +344,9 @@ protected Mediator createSpecificMediator(OMElement elem, Properties properties)
         } else {
             handleException("The collector attribute must be specified");
         }
+
+        addAllCommentChildrenToList(elem, cache.getCommentsList());
+
```

Comment:
Let's remove these unnecessary new lines in all files.

Reply:
Fixed in https://github.com/wso2/carbon-mediation/pull/1340